functionalTest/test/specs/example.e2e.js

```diff
@@ -19,53 +19,264 @@ describe("Covid-19 Dashboard", () => {
+    dashboard.healthBoard.click();
+    dashboard.cases.click();
+    expect(dashboard.healthBoard01Count).toHaveTextContaining("1995");
+    expect(dashboard.healthBoard02Count).toHaveTextContaining("549");
+    expect(dashboard.healthBoard03Count).toHaveTextContaining("464");
+    expect(dashboard.healthBoard04Count).toHaveTextContaining("1515");
+    expect(dashboard.healthBoard05Count).toHaveTextContaining("1667");
+    expect(dashboard.healthBoard06Count).toHaveTextContaining("2485");
+    expect(dashboard.healthBoard07Count).toHaveTextContaining("11165");
+    expect(dashboard.healthBoard08Count).toHaveTextContaining("747");
+    expect(dashboard.healthBoard09Count).toHaveTextContaining("5382");
+    expect(dashboard.healthBoard10Count).toHaveTextContaining("5939");
+    expect(dashboard.healthBoard11Count).toHaveTextContaining("23");
+    expect(dashboard.healthBoard12Count).toHaveTextContaining("60");
+    expect(dashboard.healthBoard13Count).toHaveTextContaining("2716");
+    expect(dashboard.healthBoard14Count).toHaveTextContaining("53");
+  });
+
+  xit("health board deaths should be displayed", () => {
+    dashboard.open();
+    dashboard.healthBoard.click();
+    dashboard.deaths.click();
+    expect(dashboard.healthBoard01Count).toHaveTextContaining("173");
+    expect(dashboard.healthBoard02Count).toHaveTextContaining("40");
+    expect(dashboard.healthBoard03Count).toHaveTextContaining("41");
+    expect(dashboard.healthBoard04Count).toHaveTextContaining("127");
+    expect(dashboard.healthBoard05Count).toHaveTextContaining("133");
+    expect(dashboard.healthBoard06Count).toHaveTextContaining("152");
+    expect(dashboard.healthBoard07Count).toHaveTextContaining("740");
+    expect(dashboard.healthBoard08Count).toHaveTextContaining("66");
+    expect(dashboard.healthBoard09Count).toHaveTextContaining("367");
+    expect(dashboard.healthBoard10Count).toHaveTextContaining("482");
+    expect(dashboard.healthBoard11Count).toHaveTextContaining("1");
+    expect(dashboard.healthBoard12Count).toHaveTextContaining("5");
+    expect(dashboard.healthBoard13Count).toHaveTextContaining("206");
+    expect(dashboard.healthBoard14Count).toHaveTextContaining("0");
+  });
+
+  // Council Areas
+  xit("council areas title text and total areas should be displayed", () => {
+    dashboard.open();
+    dashboard.councilAreas.click();
+    expect(dashboard.councilAreasTitleText).toBeDisplayed();
+    expect(dashboard.councilAreasSubheadingCount).toBeDisplayed();
+  });
+
+  xit("council areas total cases text and total count should be displayed", () => {
+    dashboard.open();
+    dashboard.councilAreas.click();
+    expect(dashboard.councilAreasTotalCasesText).toBeDisplayed();
+    expect(dashboard.councilAreasTotalCasesCount).toBeDisplayed();
+  });
+
+  xit("council areas names should be displayed", () => {
+    dashboard.open();
+    dashboard.councilAreas.click();
+    expect(dashboard.councilArea01Name).toBeDisplayed();
```

Comment:
Having enumerated methods for every row like this is a lot wordier than it needs to be. We can trim this down a lot. Either assert on the number of rows and just test the boundaries, or parameterise the councilAreaXXXName calls to not need 32 separate functions, or both :)

Reply:
This is what I expected - and what I was attempting to use for a good solution before I went with this first draft.